js/Snake.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+class SnakeHead extends Square {
+  constructor(position) {
+    super(position, 'limegreen');
+  }
+}
+
+class SnakeBody extends Square {
+ constructor(position) {
+    super(position, 'darkgreen');
+  } 
+}
+
+class Snake extends Drawable {
+  constructor(position, size) {
+    super();
+
+    this.size = size;
```

Comment:
This information is duplicated: you can also retrieve it from `this.body.length`.